src/Sfa.Das.Sas.Tools.MetaDataCreationTool/MetaDataManager.cs

```diff
@@ -207,5 +216,10 @@ private void PushStandardsToGit(List<FileContents> standards)
             _vstsService.PushCommit(standards);
             _logger.Info($"Pushed {standards.Count} new meta files to Git Repository.");
         }
+
+        public AssessmentOrganisationsDTO GetAssessmentOrganisationsData()
```

Comment:
This class is breaking single responsibility principle could you pull out GetAssementOrganisationData into another class and use Mediatr